load_env.py

```diff
@@ -1,9 +1,10 @@
 import os
 from typing import Optional
 import yaml
-
+from pathlib import Path
 from services.ssm_service import SSMService
 
+LOCAL_AWS_FILE = 'config/local-aws.yaml'
```

Comment:
Let's call these `LOCAL_SECRETS_FILE `and `local-secrets.yaml` in case there are additional secrets we need to add here.